.github/workflows/pages.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy to GitHub pages
+
+on:
+  push:
+    branches: ["master"]
+
+    # Allow running this workflow manually from the Actions tab
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+
+    steps:
+      - name: Set up directory tree
+        run: mkdir -p public_html/doc/
+      - name: Download doxygen from Linux build
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: linux.yml
+          workflow_conclusion: success
+          name: doxygen-docs
+          path: doxygen/
+      - name: Move doxygen to target directory
+        run: mv doxygen/ public_html/doc/current/
+      - name: Check out the static website
+        uses: actions/checkout@v3
+        with:
+          repository: xkbcommon/website
+          persist-credentials: false
+          path: website
+      - name: Replace absolute paths with relative ones
```

Comment:
Made this change directly in the source, so you can remove this step: https://github.com/xkbcommon/website/commit/d78f670faa163d1bffd5759d53cd1794890c3bcc

Reply:
removed, thanks!